src/main/java/org/reaktivity/nukleus/http2/internal/types/stream/Http2FrameType.java

```diff
@@ -26,7 +26,8 @@
     PING(6),
     GO_AWAY(7),
     WINDOW_UPDATE(8),
-    CONTINUATION(9);
+    CONTINUATION(9),
+    UNKNOWN(10);
```

Comment:
Suggest using `-1` for UNKNOWN instead of `10`, and therefore placing it first in the `enum`.